templates/sshd_config.erb

```diff
@@ -196,6 +196,10 @@ MaxSessions <%= @sshd_config_maxsessions_integer %>
 #MaxSessions 10
 <% end -%>
 
+<% if @sshd_config_authenticationmethods != [] -%>
+AuthenticationMethods  <%= @sshd_config_authenticationmethods.join(' ') -%>
```

Comment:
Yes and no. It accepts more than one comma separated list.
```
AuthenticationMethods  publickey,password publickey,gssapi-with-mic
```
An array of arrays could work here, but I'm not sure how to make that print in an ERB.